### ISSUE TITLE
create a mount from comand line

### ISSUE THREAD

**Issue by fantomasdm** [rhiestan/EncFSMP#8]
Hi, it's possible to create a mount from comand line or using e text file? I need a portable version of enfsmp  that can't read mount parameter from windows register.

**Comment by rhiestan** [comment]
No, sorry, this is currently not possible. You might have more luck with the original EncFS + WinFsp, see here for example:
[https://github.com/vgough/encfs/wiki/Windows](https://github.com/vgough/encfs/wiki/Windows)

By the way, no filesystem can be truly portable, it always needs to be installed because of the system drivers.
